packages/wrangler/src/index.tsx

```diff
@@ -1,5 +1,5 @@
 import * as fs from "node:fs";
-import { readFile, writeFile } from "node:fs/promises";
+import { readFile, writeFile, mkdir } from "node:fs/promises";
 import path from "node:path/posix";
```

Comment:
This is why it fails in Windows... Here you need:

```suggestion
import path from "node:path";
```

Otherwise the line below:

```ts
path.join(__dirname, **"../templates/new-worker.ts")
```

produces `templates/new-worker.ts`, which is local to the current working directory.

Reply:
Is `posix` going to be an issue in other instances of it being used? should we default to `node:path`?

Reply:
Good question. It might be safest to remove the posix usage for now.

Reply:
I looked and there are no more instances of this import.